codecov.yml

```
@@ -1,2 +1,8 @@
 comment:
   after_n_builds: 4
+
+codecov:
+  disable_default_path_fixes: true # manual fix properly covers all files
+
+fixes:
+  - "::github/" # manual fix properly covers all files
```

```suggestion
  - "::github/"
```

I don't think we need a comment here, the one above makes it clear (to me, at least)